api/controllers/TestController.js

```diff
@@ -30,7 +34,14 @@ module.exports = {
 
   loadSeasonFixture: async function (req, res) {
     try {
-      const seasons = await Season.createEach(req.body).fetch();
+      const editedSeasons = req.body.map((season) => {
```

Comment:
This feels like it warrants a comment to explain the need to transform the data. Ideally if we can write up an issue in sails we could link it here. In the meantime maybe something like this?
```suggestion
      // transform timestamps to `Date` objects, as sails-disk doesn't support ISO timestamps
      const editedSeasons = req.body.map((season) => {
```